packages/daemon/src/runner/job-runner.ts

```diff
@@ -270,4 +278,8 @@ export class JobRunner {
       }
     };
   }
+
+  async destroy(): Promise<void> {
+    await fs.remove(this.tmpdataDir);
```

Comment:
By the way, Pipboard needs to load this data to analyze, thus we shouldn't remove it.

Reply:
done